editions/tw5.com/tiddlers/filters/syntax/Cascade Filter Run Prefix (Examples).tid

```diff
@@ -0,0 +1,22 @@
+created: 20211130114857532
+modified: 20211130121656441
+tags: [[Filter Run Prefix Examples]] [[Filter Syntax]] [[Cascade Filter Run Prefix]]
+title: Cascade Filter Run Prefix (Examples)
+type: text/vnd.tiddlywiki
+
+The <<.op ":cascade">> filter run prefix is used by the ~TiddlyWiki core to choose the icon for each tiddler.
```

Comment:
here it says: "to choose the icon for each tiddler", but the examples used use the `$:/tags/TiddlerColourFilter` ... I think we should say: "to choose the tag pill colour for each tiddler" ... 

IMO "tag pill" is important, so users know where to search for the colour change if they want to test it